### ISSUE TITLE
Design Admin-Main

### ISSUE THREAD

**Issue by AnToniceQ** [SuicideSquadJecnaEdition/maturitest#4]
Main admin stránka, při requestu na /admin .

- [ ] Bez přihlášení login prompt
- [ ] Při přihlášení a admin role zobrazí všechny předměty.

**Comment by AnToniceQ** [comment]
@OndrejHana Prosím měj v paměti při vytvařování DB a backendu že je třeba nějak rozlišit ADMINA a USERA. 

Při dotazu na /admin bude mít uživatel buďto prompt na login, a nebo pokud již byl přihlášený na hlavní stránce, pouze se zkontroluje zda je přihlášený adminem, či ne, a kdyžtak mu to hodí non-admin error.

**Comment by OndrejHana** [comment]
Hádám, že `/admin` vrátí `admin.html` stránka. 
V takovém případě bude request handler kontrolovat user session a podle toho se určí jestli je uživatel autorizovaný

**Comment by AnToniceQ** [comment]
Ano, přesně tak. Ber to spíše jako upozornění, že uživatel ADMIN již může být přihlášený, jen ne přes /admin stránku.
